Binedi/Binedi.csproj

```diff
@@ -57,14 +57,16 @@
     <Compile Include="Parser.cs" />
     <Compile Include="Program.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="smtDsDict.cs" />

```

Comment:
This file is not in the PR, I guess it shouldn't be included in the project file, right?